packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedFieldMessageEmbed.ts

```diff
@@ -0,0 +1,155 @@
+import { MessageEmbed, MessageEmbedOptions } from 'discord.js';
+
+import { PaginatedMessage } from './PaginatedMessage';
+
+/**
+ * This is a utility of {@link PaginatedMessage}, but paginating only the field of an embed.
+ * You must either use this class directly or extend it.
+ *
+ * @example
+ * ```typescript
+ * const { PaginationFieldMessageEmbed } = require('@sapphire/discord.js-utilities');
+ *
+ * new PaginationFieldManager()
+ *    .setTitleField('')
+ *    .setTemplate({ ...options })
+ *    .setItems([{ field: 'value' }, { field: 'value' }, { field: 'value' }, { field: 'value' }])
```

Comment:
Why is this an array of objects when each object only has 1 key? Could also just make an array of strings then.

Reply:
It is an example of use.

Reply:
I get that it is an example but like this the syntax is very weird. So either show some other examples or do something about the syntax.

Reply:
But I don't see what's confusing, I'm just adding an array.